src/main/java/org/broadinstitute/hellbender/utils/gcs/BucketUtils.java

```diff
@@ -66,7 +68,8 @@ public static boolean isCloudStorageUrl(final GATKPathSpecifier pathSpec) {
 
     public static boolean isCloudStorageUrl(final java.nio.file.Path path) {
         // the initial "" protects us against a null scheme
-        return ("" + path.toUri().getScheme() + "://").equals(GCS_PREFIX);
+        final String prefix = "" + path.toUri().getScheme() + "://";
+        return prefix.equals(GCS_PREFIX) || prefix.equals(HTTP_PREFIX) || prefix.equals(HTTPS_PREFIX);
```

Comment:
I think that `isCloudStorageUrl()` is meant to test specifically for GCS urls, as the name implies, so I don't think we should overload it to also check for http. Recommend introducing a new method `isHTTPUrl()`, and calling it from appropriate places such as `BucketUtils.isRemoteStorageUrl()`

Reply:
oh, yeah, that was the patch I was working on that I delayed and then forgot to push.

Reply:
I'm reshuffling things a bit.  It's not clear to me what "cloud storage" means exactly.  I examined the different uses of these methods and it looks like some overloads are used to test "is this a google url" and need to remain google only.  I'm renaming those to `isGcsUrl` which is clear that it's GOOGLE cloud storage. This overload is only used in places where you want to know if it's some sort of laggy remote file.  I'm renaming this to "isEligibleForPrefetching" to make it clear what it's asking.